wagtail/admin/templates/wagtailadmin/shared/_workflow_init.html

```diff
@@ -1,36 +1,44 @@
 <script>
     $(function() {
-        ActivateWorkflowActionsForEditView('[data-edit-form]');
+        document.querySelectorAll('footer [data-controller="w-dropdown"]').forEach((e) => {
+            e.addEventListener(
+                'w-dropdown:shown',
+                () => {
```

Comment:
Similar in principle to https://github.com/wagtail/wagtail/pull/11093/files?w=1#r1365704925, with a few notes:

- The jQuery `$(function() {` wrapper is still necessary (unless we replace it with a `DOMContentLoaded` listener) because the `extra_js` block is rendered before the main page content is rendered, so the script would execute immediately. (In fact, I wonder why the code I linked above works without it...)
- Selecting the `w-dropdown` inside `footer` specifically as there could be other dropdowns on the page (header button, locale selector, etc.). Going against the guidelines since I use something other than a `data-` attribute (`footer`) to attach this, but the alternative is probably the `.footer` class which might be refactored into `.w-footer` or something when #9030 is implemented. Ideally we'd probably add another `data-` attribute, but I think it isn't worth it now.

Reply:
Some of this will resolve itself once we move workflow actions to Stimulus, but that requires some rework or rethinking of the `ModalWorkflow` first. Which also requires a look at the dialogs used there :D 

We will get there though.

Reply:
Yep exactly – one step at a time 😄